playwright-tests/tests/settings/create-member-request.spec.js

```diff
@@ -575,4 +593,45 @@ test.describe("User is logged in", function () {
     await expect(loader).toBeHidden();
     await expect(submitBtn).toBeEnabled();
   });
+
+  test("should show warning when adding member having non complete proposals", async ({
```

Comment:
can you please attach the video of the test, and also add another test where there is no existing proposal (or maybe we already have it), and user doesn't see the modal.

and also add a similar test for voting threshold since we have made the change in that page as well (w and w/o modal)

Reply:
[video.webm](https://github.com/user-attachments/assets/8d8b7824-8d1f-4130-b565-26ca3fa03bf3)
[video.webm](https://github.com/user-attachments/assets/7f73a43b-3e06-4d81-ac85-df9d7b208b2f)

[video.webm](https://github.com/user-attachments/assets/e01ed691-5c88-4733-b6c0-cb50a86e1b89)
[video.webm](https://github.com/user-attachments/assets/24812f3a-4a64-415b-a697-44db1dbeefb7)